static/js/filter.js

```diff
@@ -1,7 +1,95 @@
 (function() {
 	"use strict";
-
     
-	
-	
-})();
\ No newline at end of file
+	const filterEmailAddresses = [];
+	const filterAliasLabels = [];
+	const aliasesWithLabelsCollection = [];
+    const aliasCollection = [];
```

Comment:
preceding lines are indented with 2 spaces, these are 4?

Reply:
Actual answer: We need a linter™.